ui/src/app/applications/components/pod-logs-viewer/pod-logs-viewer.tsx

```diff
@@ -91,8 +91,6 @@ export const PodsLogsViewer = (props: PodLogsProps) => {
     const logsContainerRef = useRef(null);
 
     const setWithQueryParams = <T extends (val: any) => void>(key: string, cb: T) => {
-        history.replaceState(null, '', `${location.pathname}?${queryParams}`);
-
```

Comment:
Do you have any sense why this was present in the first place? Can we be confident that it's not needed?

Reply:
Ah nevermind the original author gave it the thumbs-up. @linghaoSu how far back do we need to cherry-pick this?

Reply:
I have checked the commit this bug introduced, this change does not ship in any RC or stable version. We don't need to cherry-pick, merging this into the master branch is enough. 

![image](https://github.com/user-attachments/assets/0ad51526-11f4-49d0-a9d4-b8567181922a)

Reply:
Solves it for me.